rcl/test/rcl/test_get_actual_qos.cpp

```diff
@@ -249,7 +357,10 @@ expected_system_default_subscription_qos_profile()
   rmw_qos_profile_t profile = rmw_qos_profile_default;
   profile.depth = 1;
   profile.reliability = RMW_QOS_POLICY_RELIABILITY_BEST_EFFORT;
+  profile.deadline.sec = 2147483647;
+  profile.lifespan.sec = 2147483647;
```

Comment:
Delete line, as lifespan is not used in subscriptions

Reply:
Done.